src/devices/Fsr408/Fsr408.cs

```diff
@@ -0,0 +1,154 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+using System.Diagnostics;
+
+namespace Iot.Device.Fsr408
+{
+    public class Fsr408 : IDisposable
+    {
+        private GpioController _controller;
+        private int _pinNumber = 18;
+
+        public Mcp3008.Mcp3008 AdcConverter { get; set; }
+        public int Resistance { get; set; } = 10000; // 10k ohm
+        public int PowerSupplied { get; set; } = 5000;  // 5 mV
+
+        public Fsr408()
```

Comment:
Yeah well, there is another optional paramtere _pinNumber which could be set using below constructor, adding another optional constractor parameter lead to creating 4 constructors with parameter combinations, felt its better make Adc converter as public property, if you prefer constructor then would change

Reply:
Would you always need an Mcp3008 in order for the device to work? I see methods like `ReadFromMcp` and others for ReadingVoltage and resistance that seem to really depend on an Mcp being defined. If that's the case, I would just add the mcp constructor since it seems to be something really essential for interacting with the sensor.

Reply:
`Would you always need an Mcp3008 in order for the device to work?`
No it can work with a plain capacitor too. I might better to split this 2 implementations ...

Reply:
I think until we have generic AdcConverter class this is fine as is. I'd personally merge this as is, then add AdcConverter, once everyone agrees on the design we can fixup the implementations.

@buyaa-n is it possible that the capacitor acts as ADC in this case? If so we could perhaps add it as a device (PoorManAdc with some better name :smile:)

Reply:
> No it can work with a plain capacitor too. I might better to split this 2 implementations ...

In this case I wouldn't have methods that specifically call out the `FromMcp3008` since then you are leaking the connectivity type to the Api. Instead, I would have a method called Read(), which inside has something like
```c#
if (_usingMcp)
{
    // Read using Mcp
}
else 
{
    // Read using capacitor
}
```

That way the caller doesn't need to know how the device is connected, and the only difference is that they would use a different constructor if they are using an Mcp or if they are using a capacitor. Does that make sense?

Reply:
Yeah I had something like that at first, but the problem is resulting read values are not match. I mean read value from ADC and capacitor is different. For capacitor its producing charging duration with lots of noise, not accurate or not stable values, pretty much seems to me its appropriate only for usage to sense if FSR is pressed or not. But ADC gives accurate sampled values from which we can calculate read voltage, resistance and pressure force.

Reply:
In that case, if capacitor really doesn't produce any valuable information, I would just not support it at all and only work with an ADC. In any case, if you do that then I would still just remove the Mcp3008 from your API methods.

Reply:
Basically, I would expect that your public surface area be something like:

```c#
int ReadVoltage();
int ReadResistance();
double ReadForce();
```

with each of those explaining in the comments which units will be given back for each of the methods.

Reply:
I'd go with properties, you are not reading, you are converting analog signal